src/lib/RequestParser.ts

```diff
@@ -79,13 +90,19 @@ export class RequestParser {
             case RequestType.LOGOUT:
             case RequestType.ADD_ADDRESS:
                 const simpleRequest = request as SimpleRequest;
+
+                if (!simpleRequest.accountId) throw new Error('accountId is required');
```

Comment:
I think this should also be enforced for `SIGN_TRANSACTION` and `CHECKOUT`. Probably `appName`, too.

Reply:
`CHECKOUT` does not use an `accountId`, for `SIGN_TRANSACTION` and `SIGN_MESSAGE` is was removed earlier in #197.

Only question is if we should check for an `appName`.

Reply:
I added a check for the appName to the top, so it is checked for all requests.